src/sagas/task.saga.js

```diff
@@ -64,19 +64,10 @@ export function* launchDevServer({ task }: Action): Saga<void> {
 
     const stdioChannel = createStdioChannel(child, {
       stdout: emitter => data => {
-        // Ok so, unfortunately, failure-to-compile is still pushed
-        // through stdout, not stderr. We want that message specifically
-        // to trigger an error state, and so we need to parse it.
-        const text = data.toString();
-
-        // Ok, so this workflow really needs to be rewritten.
-        // We can't safely assume that all `stderr` output indicates
-        // a breaking/blocking error, since Jest writes its standard
-        // output to `stderr` (presumably to keep it from displaying in
-        // CI logs). Currently the only thing we know for sure is a
-        // breaking error is when the dev server fails to compile, so we're
-        // specifically handling that case, but we need a better, more
-        // generalized solution.
```

Comment:
I ditched this comment because it wasn't quite right; this is referring to stderr stuff on test tasks, but this is the stdout for the dev-server task specifically.